debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/UniqueDatabase.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class UniqueDatabase {
+    private static final String DEFAULT_DATABASE = "mysql";
+    private static final String[] CREATE_DATABASE_DDL = new String[] {
+            "CREATE DATABASE $DBNAME$;",
+            "USE $DBNAME$;"
+    };
+
+    private final String databaseName;
+    private final String templateName;
+    private final String serverName;
+
+    public UniqueDatabase(final String databaseName, final String serverName) {
+        this.databaseName = databaseName + "_" + Integer.toUnsignedString(new Random().nextInt(), 36);
```

Comment:
Ah, this explains the "Unique" in the name. That wasn't quite clear to me first.